mtp_cashbook/templates/disbursements/hand-over.html

```diff
@@ -0,0 +1,58 @@
+{% extends 'disbursements/base.html' %}
+{% load i18n %}
+{% load mtp_common %}
+
+{% block body_classes %}{{ block.super }} mtp-create-disbursement{% endblock %}
+
+{% block inner_content %}
+  {{ block.super }}
+
+  <header>
+    <h1 class="heading-xlarge">{% trans 'Hand over the paper form for confirmation' %}</h1>
+  </header>
+
+  {% if errors %}
+    <div class="error-summary" aria-labelledby="error-summary-heading" tabindex="-1" role="alert">
+      <h2 class="heading-small error-summary-heading" id="error-summary-heading">
+        {% trans 'There was a problem' %}
+      </h2>
+      <ul class="error-summary-list">
+        {% for error in errors %}
+          <li class="non-field-error">{{ error }}<span class="visually-hidden">.</span></li>
+        {% endfor %}
+      </ul>
+    </div>
+  {% endif %}
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+
+      <p>
+        {% trans 'Ask a colleague to double-check the online details match those on the paper form.' %}
+      </p>
+      <p>
+        {% trans 'To do this, hand your colleague the paper form.' %}
+        {% trans 'Then ask them to go to ‘Confirm payments’ in the digital disbursements tool.' %}
+      </p>
+      <p>
+        {% trans 'Anyone on your team who has an account on ‘Manage prisoner money’ can confirm the payment.' %}
+      </p>
+
+      {% random_string as note_id %}
+      <p>
+        <a class="mtp-disclosure" href="#" aria-controls="mtp-{{ note_id }}" aria-expanded="false" role="button" data-analytics="pageview,/-help_open/{{ request.resolver_match.url_name }}/">
+          <span></span>{% trans 'What if my colleague doesn’t have an account?' %}
+        </a>
+      </p>
+      <div id="mtp-{{ note_id }}" class="mtp-disclosure__contents panel panel-border-narrow">
+        <p>
+          {% trans 'Ask your business hub manager to set one up.' %}
+          {% trans 'Boom!' %}
```

Comment:
i hate this

Reply:
I will not be merging this until it is removed